src/client/App.js

```diff
@@ -11,20 +11,33 @@ import Profile from './containers/Profile';
 import Loader from './components/Loader';
 import NotFoundPage from './containers/NotFound/notFoundPage.component';
 import Footer from './components/Footer/Footer.js';
+import { CVreviews } from './components/CVreviewsPage/CVreviews';
+import { AddResume } from './components/AddResume/AddResume';
 
 function App() {
+  const [uploadedFile, setUploadedFile] = useState('');
   const { isAuthenticated, isLoading } = useAuthentication();
   if (isLoading) return <Loader />;
   return (
     <Router>
       {location.pathname === '/sign-in' ||
       location.pathname === '/sign-up' ? null : (
-        <Header isAuthenticated={isAuthenticated} username="William Henry Gates" />
+        <Header
+          isAuthenticated={isAuthenticated}
+          username="William Henry Gates"
+        />
       )}
       <Switch>
         <Route exact path="/">
           <Home />
         </Route>
+        <AddResume
```

Comment:
this shouldn't be in app.js. this component should be in my profile page and when you click on a button (upload)then this should be displayed.

Reply:
yes you're right. I think it is not about my story. but actually i needed a place to define two siblings AddResume and CVreviews and uploadedFile state. because in both of them I should use uploadedFile. I am not sure who is responsible for making my profile page functionality.

Reply:
yes, but i think you shouldn't push app.js changes. because if it merged into develop everyone gets this update and it'll be hard to add other relative components

Reply:
@hamedveloper  it's better not to apply this component to any page right now, we invoke it from profile later.